### DISCUSSION TITLE
Account Key Rollover Support?

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [acmesh-official/acme.sh#4282] [Q&A] [answered]
Account Key Rollover? https://www.rfc-editor.org/rfc/rfc8555#section-7.3.5

**Comment by neilpang** [comment, marked as answer]
what do you mean for?

**Reply by IzumiSenaSora** [reply]
[RFC](https://www.rfc-editor.org/rfc/rfc8555#section-7.3.5) Account Key Rollover 
A client may wish to change the public key that is associated with an
account in order to recover from a key compromise or proactively
mitigate the impact of an unnoticed key compromise.

**Reply by neilpang** [reply]
Sorry, it's not implemented yet.

**Reply by IzumiSenaSora** [reply]
Will you add this feature in future?

**Reply by neilpang** [reply]
It depends on the user demands.

**Reply by miken32** [reply]
I assume no change in this? I couldn't find anything in the command arguments to suggest it's been added.
